crates/bevy_winit/src/lib.rs

```diff
@@ -142,12 +144,14 @@ impl<T: Event> Plugin for WinitPlugin<T> {
 #[derive(Debug, Default, Clone, Copy, Event)]
 pub struct WakeUp;
 
-/// The [`winit::event_loop::EventLoopProxy`] with the specific [`winit::event::Event::UserEvent`] used in the [`winit_runner`].
+/// A wrapper type around [`winit::event_loop::EventLoopProxy`] with the specific
+/// [`winit::event::Event::UserEvent`] used in the [`winit_runner`].
 ///
 /// The `EventLoopProxy` can be used to request a redraw from outside bevy.
 ///
 /// Use `NonSend<EventLoopProxy>` to receive this resource.
```

Comment:
Does this comment still holds truth after the changes? (`"Use Res<...>"`)

Reply:
That comment should be updated. Nice catch.